nitro/Dockerfile

```diff
@@ -20,43 +20,38 @@ RUN apt-get update && \
     apt-get install --no-install-recommends -y \
         apt-transport-https \
         ca-certificates \
-        gnupg \
-        iproute2 \
-        iptables \
-        jq \
         lsb-release \
         lxc \
-        musl-tools \
-        python3 \
-        python3-pip \
-        python3-setuptools \
-        software-properties-common ; \
-    curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add - ; \
-    add-apt-repository "deb https://download.docker.com/linux/ubuntu bionic stable" ; \
-    apt-get update && apt-get install --no-install-recommends -y docker-ce-cli ; \
-    apt-get autoremove -y && apt-get clean ; \
+        musl-tools && \
+    curl -fsSL https://download.docker.com/linux/ubuntu/gpg | \
```

Comment:
Why do we need GPG inside the docker container?

Reply:
Because we need to add the Docker GPG key for their repository and we also verify the integrity of `repo`